src/Symfony/Component/Validator/Resources/translations/validators.sl.xlf

```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Vrednost omrežne maske mora biti med {{ min }} in {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znak ali manj.|Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znaka ali manj.|Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znake ali manj.|Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znakov ali manj.</target>
```

Comment:
Just checking. 

The English line has two translations, one for singular and one for plural. 
```
The filename is too long. It should have {{ filename_max_length }} character or less.
The filename is too long. It should have {{ filename_max_length }} characters or less.
```

What I can see, there are 4 different strings. Are these the different plural in Slovenian?

```
Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znak ali manj.
Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znaka ali manj.
Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znake ali manj.
Ime datoteke je predolgo. Imeti mora {{ filename_max_length }} znakov ali manj.
```

Reply:
@Nyholm Slovenian indeed uses 4 plural forms: https://github.com/symfony/symfony/blob/3e9cabfbae5376700a5ec7cba749cb656966e683/src/Symfony/Contracts/Translation/TranslatorTrait.php#L214